x-pack/plugins/apm/public/components/routing/templates/apm_service_template/index.tsx

```diff
@@ -404,7 +404,10 @@ function useTabs({ selectedTab }: { selectedTab: Tab['key'] }) {
       label: i18n.translate('xpack.apm.home.profilingTabLabel', {
         defaultMessage: 'Universal Profiling',
       }),
-      hidden: !isProfilingAvailable,
+      hidden:
```

Comment:
shouldn't be the other way around: 
`!isProfilingAvailable || isMobileAgentName(agentName) || isRumAgentName(agentName)` ?

Reply:
oh yes... I think I'm still 😴 good catch!